### ISSUE TITLE
Disable path regularization and lazy regularization

### ISSUE THREAD

**Issue by nupurkmr9** [mit-han-lab/data-efficient-gans#57]
Hi, 
In the paper, for FFHQ 1k 256x256 training with DiffAugment, it is written that path length regularization and lazy regularization is disabled. If I am not wrong, in the DiffAugment-stylegan2-pytorch repo lazy regularization and path length regularization is still there, right? 
Just wanted to confirm this before I start any training. :)
Thanks!

**Comment by zsyzzsoft** [comment]
Yes. You may need to change them and possibly some other hyperparameters as said in the paper, to fully reproduce our results in the TensorFlow version.

**Comment by nupurkmr9** [comment]
Thanks. Can you tell what other hyperparameters need to be changed for FFHQ with config "paper256"?

**Comment by zsyzzsoft** [comment]
mb=32, mbstd=4, lrate=0.002, and enable the mirror augmentation

**Comment by nupurkmr9** [comment]
Thanks a lot!!
Also, on calculating FID on the provided model "DiffAugment-stylegan2-ffhq-1k.pkl" with a real dataset of 1K images from FFHQ, it comes out to be somewhere close to 41. I used the calc_metrics script and the real images were png images downloaded from google drive folder provided at https://github.com/NVlabs/ffhq-dataset. Just wanted to confirm if this behavior is expected since I am comparing with random 1K images and not the training set?
